src/main/java/picard/vcf/UpdateVcfSequenceDictionary.java

```diff
@@ -89,7 +91,15 @@ protected int doWork() {
         if (CREATE_INDEX)
             builder.setOption(Options.INDEX_ON_THE_FLY);
 
-        final VariantContextWriter vcfWriter = builder.setOutputFile(OUTPUT).build();
+        // handle stdout
+        if ( OUTPUT.getPath().equals(STD_OUT) ) {
```

Comment:
I don't think you need the if statement here at all, as @nh13 said. please remove it and wrap OUTPUT with a FileOutputStream (and remove the STD_OUT constant)